chcemvediet/apps/accounts/admin.py

```diff
@@ -79,3 +89,20 @@ def get_queryset(self, request):
         queryset = queryset.select_related(u'user')
         queryset = queryset.select_undecided_emails_count()
         return queryset
+
+    def login_as_view(self, request, id):
```

Comment:
Pri model admin views zvykneme "id" parameter volat `obj_pk`. Lebo realne to nemusi byt nic s nazvom "id".

Reply:
:ok: